roles/srv-announcer/tasks/configure.yml

```
@@ -15,12 +15,11 @@
     mode: '600'
   register: unit_conf_srva
 
-- name: copying systemd service unit(es) to announce SRV record(s)
+- name: copying systemd service unit to announce SRV record(s)
   template:
     src: 'srv-announcer.service.j2'
-    dest: "/etc/systemd/system/{{ srv_announcer_service_name }}.{{ item.key }}.service"
+    dest: "/etc/systemd/system/{{ srv_announcer_service_name }}.service"
     owner: root
     group: root
     mode: '644'
-  loop: "{{ srv_announcer_srv_records | dict2items }}"
-  register: units_srva
+  register: unit_srva
```

```suggestion
  register: unit_srv_announcer
```

Unrelated, but I stumbled it over during review. Making this string a bit longer doesn't hurt, especially considering how many other characters we remove ;-)

---

[Addressed](https://github.com/wireapp/ansible-sft/pull/8/commits/8ec9040f0dc18c921b5f2faa6170af6c55b4728b).